Source/WebKit/Shared/ApplePay/cocoa/PaymentTokenContextCocoa.mm

```diff
@@ -38,13 +39,6 @@
 namespace WebKit {
 using namespace WebCore;
 
-static NSDecimalNumber *toDecimalNumber(const String& amount)
-{
-    if (!amount)
-        return NSDecimalNumber.zero;
-    return [NSDecimalNumber decimalNumberWithString:amount locale:@{ NSLocaleDecimalSeparator : @"." }];
-}
-
```

Comment:
Is this change necessary in this patch? I see a couple of other places that have this code duplicated, so maybe it's better we just address it in a follow-up.

Reply:
It was necessary here, because I'm adding files, which changes how files are unified, causing this one to start clashing with another one! It can't be a follow-up, but it could be in a prior one, if you really wanted this change out of this patch.

Reply:
I see! This is fine for now. We can always do a follow-up afterwards.